vitest.config.ts

```diff
@@ -8,6 +8,7 @@ export default mergeConfig(
       sourcemap: "inline",
     },
     test: {
+      environment: "jsdom",
```

Comment:
what does this do?

Reply:
I did face crashes on testing functions with custom CSS styles from react-components in the default environment `node`. The change to `jsdom` is emulating a more realistic browser environment. This resolved this issue. See also [vite test docs](https://vitest.dev/guide/environment) on this.